backend/canisters/user/CHANGELOG.md

```diff
@@ -13,13 +13,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - Update group chat summary cache in small batches ([#3341](https://github.com/open-ic/open-chat/pull/3341))
+- Cannot remove platform moderators from a group ([#3340](https://github.com/open-ic/open-chat/pull/3340))
```

Comment:
It isn't very clear what this change is. It's hard to tell if its making it so you cannot remove them or if it is fixing the fact that you couldn't remove them.

Maybe 'prevent' or 'disable' would be better.